index.html

```diff
@@ -33,6 +33,127 @@ <h2 class="name">I’m Ali<br>Glad to see you!</h2>
             </ul>
         </div>
       </article>
+      <!-- here we start the second milestone -->
+      <section class="grid-container">
+        <article class="middle tonic">
+          <h1 class="middle-h1">Tonic</h1>
+          <div class="canopy">
+            <span class="canopy-1">CANOPY</span>
+            <img src="images/Counter.png" alt="">
+            <span class="canopy-2">Back End Dev</span>
+            <img src="images/Counter.png" alt="">
+            <span class="canopy-2">2015</span>   
+          </div>
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio.png" alt="Snapshoot Portfolio" class="middle-images">
+          <div class="middle-button">
+            <button class="html-button">html</button>
+            <button class="css-button">css</button>
+            <button class="js-button">javaScript</button>
+          </div>
+          <button type="button" class="button-see-project">See Project</button>
+        </article>
+        <article class="middle multi-post-stories">
+          <h1 class="middle-h1">Multi-Post Stories</h1>
+          <div class="canopy">
+            <span class="canopy-1">CANOPY</span>
+            <img src="images/Counter.png" alt="">
+            <span class="canopy-2">Back End Dev</span>
+            <img src="images/Counter.png" alt="">
+            <span class="canopy-2">2015</span>   
+          </div>
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio2.png" alt="Snapshoot Portfolio 2" class="middle-images">
+          <div class="middle-button">
+            <button class="html-button">html</button>
+            <button class="css-button">css</button>
+            <button class="js-button">javaScript</button>
+          </div>
+          <button type="button" class="button-see-project">See Project</button>
+        </article>
```

Comment:
- [ ]The list of projects should use a` ul tag` to group them, and `li` per project